package.json

```diff
@@ -49,14 +49,15 @@
     "trace": "MODE=trace truffle test",
     "coverage": "yarn generate && ./scripts/run-coverage",
     "clean-coverage": "shx rm -rf coverage coverage.json",
-    "generate": "rm -rf ./build/contracts && truffle compile && typechain --target trufflehotfix './build/contracts/**/*.json'",
+    "generate": "rm -rf ./build/contracts && npx truffle compile && typechain --target trufflehotfix './build/contracts/**/*.json'",
```

Comment:
Do you need 'npx'? I was under impression npm knows to look in your node modules for commands